.travis.yml

```diff
@@ -22,11 +22,9 @@ install:
   - make setup
 
 env:
-  - RUN="make lint"
-  - RUN="make test-unit"
+  - RUN="make test-unit test-examples lint coveralls"
```

Comment:
This is part of #181 - If you want changes on it please file them there. TL;DR it speeds up a complete travis matrix run a lot.